### ISSUE TITLE
Nothing happens after install

### ISSUE THREAD

**Issue by oblitum** [UltiRequiem/coc-zig#1]
Recently I tried this plugin but it did nothing, it didn't load at all. I went back to server configuration.

**Comment by UltiRequiem** [comment]
> Recently I tried this plugin but it did nothing, it didn't load at all. I went back to server configuration.

Could you provide the way how you installed coc + nvim version + `:CocInfo`, please?

The last time I checked it was working correctly. I don't code in zig in a long time tho.

**Comment by oblitum** [comment]
I'm on Arch Linux, currently using nvim nigthly:

```
## versions

vim version: NVIM v0.7.0-dev+1275-g00effff56
node version: v17.8.0
coc.nvim version: 0.0.80-5830e03d 2022-03-27 04:58:12 +0800
coc.nvim directory: ~/.local/share/nvim/site/pack/packer/opt/coc.nvim
term: tmux
platform: linux
```

`zig` was installed from Arch repos, and `zls-bin` from AUR. I tried to set `zig.path` but it did nothing too. Using bare language server config for `zls` works without problems.

**Comment by shmuga** [comment]
Same happened to me. I have introduced the fix in #3 

There were two main issues: config parameter had a type and default export was used. This resulted in plugin been not activated.
